client/src/app/core/repositories/motions/motion-change-recommendation-repository.service.ts

```diff
@@ -15,6 +16,18 @@ import { RepositoryServiceCollector } from '../repository-service-collector';
 import { ViewMotion } from '../../../site/motions/models/view-motion';
 import { ViewUnifiedChange } from '../../../shared/models/motions/view-unified-change';
 
+export interface RawMotionChangeRecommendation {
```

Comment:
Can this be moved to `motion-change-reco.ts` and let the `MotionChangeRecommendation` extend this interface. BTW. please rename the file to `motion-change-recommendation.ts`.

Reply:
This interface is also very similar to `MotionChangeRecommendation`. Maybe just use `MotionCHangeRecommendation` or the `Partial<..>`-version of it.